privacyidea/api/lib/utils.py

```diff
@@ -148,6 +148,12 @@ def send_error(errstring, rid=1, context=None, error_code=-311, details=None):
     return ret
 
 
+def send_file(output, filename):
```

Comment:
For the final PR, could you add a small docstring?

Reply:
Done in ba5ab0b